pkg/planner/core/plan_cost_ver2.go

```diff
@@ -103,6 +103,17 @@ func (p *PhysicalProjection) GetPlanCostVer2(taskType property.TaskType, option
 	return p.PlanCostVer2, nil
 }
 
+const (
+	// MinNumRows provides a minimum to avoid underestimation
+	MinNumRows = 1.0
+	// MinRowSize provides a minimum to avoid underestimation
+	MinRowSize = 2.0
```

Comment:
Agree with your reason to set these 2 minimum for non-index based operations, but could you add code comments here to explain why we want to add them? For example, just moving some comments in the PR description to here as code comments.

Reply:
Done